<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SHSL Editor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabs">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tabsPage1">
       <attribute name="title">
        <string>Project Info</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QListView" name="projectInfo"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabsPage2">
       <attribute name="title">
        <string>Game Files</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QSplitter" name="splitter_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QSplitter" name="splitter">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <widget class="QListWidget" name="wadList">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="gridSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
           </widget>
           <widget class="QWidget" name="verticalLayoutWidget">
            <layout class="QVBoxLayout" name="verticalLayout">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <property name="spacing">
                <number>0</number>
               </property>
               <item>
                <widget class="QLineEdit" name="wadFileTreeFilter">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="placeholderText">
                  <string>Filter</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QToolButton" name="wadFileTreeFilterReset">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QTreeView" name="wadFileTree">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="contextMenuPolicy">
                <enum>Qt::CustomContextMenu</enum>
               </property>
               <property name="editTriggers">
                <set>QAbstractItemView::NoEditTriggers</set>
               </property>
               <property name="uniformRowHeights">
                <bool>true</bool>
               </property>
               <attribute name="headerVisible">
                <bool>false</bool>
               </attribute>
               <attribute name="headerStretchLastSection">
                <bool>false</bool>
               </attribute>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
          <widget class="QTabWidget" name="wadFileTabs">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="currentIndex">
            <number>1</number>
           </property>
           <widget class="QWidget" name="wadFileHexTab">
            <attribute name="title">
             <string>Hex View</string>
            </attribute>
            <layout class="QGridLayout" name="gridLayout_2"/>
           </widget>
           <widget class="QWidget" name="wadFileImageTab">
            <attribute name="title">
             <string>Image View</string>
            </attribute>
            <layout class="QGridLayout" name="gridLayout"/>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew_Project"/>
    <addaction name="actionOpen_Project"/>
    <addaction name="separator"/>
    <addaction name="actionClose_Project"/>
    <addaction name="separator"/>
    <addaction name="actionSet_Game_Directory"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew_Project">
   <property name="text">
    <string>&amp;New Project...</string>
   </property>
   <property name="shortcut">
    <string>N</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetShortcut</enum>
   </property>
  </action>
  <action name="actionOpen_Project">
   <property name="text">
    <string>&amp;Open Project...</string>
   </property>
   <property name="shortcut">
    <string>O</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetShortcut</enum>
   </property>
  </action>
  <action name="actionClose_Project">
   <property name="text">
    <string>Close Project</string>
   </property>
  </action>
  <action name="actionSet_Game_Directory">
   <property name="text">
    <string>Set &amp;Game Directory...</string>
   </property>
   <property name="shortcut">
    <string>G</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetShortcut</enum>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>projectInfo</tabstop>
  <tabstop>wadFileTabs</tabstop>
  <tabstop>wadList</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
